### ISSUE TITLE
blocktrail.com is dead

### ISSUE THREAD

**Issue by Mullick** [blocktrail/wallet-recovery-tool#23]
```

Address not found
Your message wasn't delivered to support@blocktrail.com because the address couldn't be found, or is unable to receive mail.
```

Where can you get the decryption key if you forgot the password of a v2/v3 wallet now?

**Comment by DanielDeng3** [comment]
even you remember you password, you can't get recovey wallet, it's need to link to back server "recovery-backend.blocktrail.com/blocktrail-publickey", but this website is unreachable, so fuck this project, fuck btc.com !

**Comment by 3rdIteration** [comment]
> even you remember you password, you can't get recovey wallet, it's need to link to back server "recovery-backend.blocktrail.com/blocktrail-publickey", but this website is unreachable, so fuck this project, fuck btc.com !

You actually can as the wallet backup PDF will include the blocktrail pubkeys you need.

**Comment by 3rdIteration** [comment]
> ```
> 
> Address not found
> Your message wasn't delivered to support@blocktrail.com because the address couldn't be found, or is unable to receive mail.
> ```
> 
> Where can you get the decryption key if you forgot the password of a v2/v3 wallet now?

I added password recovery support for for v3 wallets to BTCRecover here: https://github.com/3rdIteration/btcrecover so if you have some idea about what the password was, you may be able to work it out with the help of BTCRecover.

**Comment by DanielDeng3** [comment]
> > even you remember you password, you can't get recovey wallet, it's need to link to back server "recovery-backend.blocktrail.com/blocktrail-publickey", but this website is unreachable, so fuck this project, fuck btc.com !
> 
> You actually can as the wallet backup PDF will include the blocktrail pubkeys you need.
yes i know ，but the offical [recovery.blocktrail.com] can't reach out to the sever， i think  it need  to contact  server  for decode the encrypted seed，but you know  btc.com not run their server anymore ，so can't to decode.
